app/client/cypress/integration/state/water-quality-overview/subTab.spec.js

```diff
@@ -0,0 +1,66 @@
+describe('Water Quality Overview Sub Tabs', () => {
+  beforeEach(() => {
+    cy.visit('/state/FL/water-quality-overview');
+  });
+
+  it('Navigating to a sub-tab selection that has no data results in “Water Type” dropdown saying “No Available Water Types” and the “Use” dropdown saying “No Available Uses”', () => {
+    // verify the water quality overview content loaded prior to other tests
+    cy.findByText('Water Quality', { timeout: 20000 }).should('exist');
+
+    // wait for the all web services to finish (surveys is usually slow here)
+    cy.findAllByTestId('hmw-loading-spinner', { timeout: 20000 }).should(
+      'not.exist',
+    );
```

Comment:
Since you're doing this in both tests, you could move this up to the `beforeEach()` block, after the `cy.visit()` call.

Reply:
Good call!